pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Check your pom.xml and remove all dependencies that you never used.  h2 for example